kong/plugins/azure-functions/handler.lua

```diff
@@ -97,15 +71,15 @@ function azure:access(config)
 
   local ok, err = client:connect(config.host, config.port)
   if not ok then
-    ngx_log(ngx.ERR, "[azure-functions] could not connect to Azure service: ", err)
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+    kong.log.err("could not connect to Azure service: ", err)
+    return kong.response.exit(500, { msg = "An unexpected error ocurred" })
```

Comment:
Why is it `msg`, shouldn't it be `message`?

Reply:
Lack of sleep does this. xD